packages/ui/v2/core/layouts/BookingLayout.tsx

```diff
@@ -45,10 +45,10 @@ export default function BookingLayout({
     <Shell {...rest}>
       <div className="flex flex-col sm:space-x-2 xl:flex-row">
         <div className="hidden xl:block">
-          <VerticalTabs<"tabName"> tabs={tabs} sticky />
+          <VerticalTabs tabNameKey="tabName" tabs={tabs} sticky />
```

Comment:
Minor nit - should tabName not be the default? and we only override if tabNameKey value is passed int?